builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -1064,12 +1064,11 @@ def createWebsite(stackName, resourceGroup, storageName, framework){
       storageName       : storageName,
       appName           : stackName,
       tags: [
-        "application"   : configLoader.INSTANCE_PREFIX,
-        "owner"         : service_config['created_by'],
-        "domain"        : service_config['domain'],
-        "STAGE"         : environment_logical_id,
-        "environment"   : environment_logical_id,
-        "service"       : stackName
+        "Application"   : configLoader.INSTANCE_PREFIX,
```

Comment:
In the serverless yml templates and in distribution_config_with_tags.json  "Application" mentioned as "Jazz" and here it is instance prefix. Also not every key is present here. Is that expected?

Reply:
Fixed.